src/components/ProfileHeader.jsx

```diff
@@ -0,0 +1,95 @@
+import User1 from "../assets/user/user1.svg";
+import {
+  DotsThree,
+  CaretLeft,
+  CaretDown,
+  Envelope,
+} from "@phosphor-icons/react/dist/ssr";
+import useMediaQuery from "../hooks/useMediaQuery";
+
+const ProfileHeader = ({ user }) => {
+  const isAboveMediumScreens = useMediaQuery("(min-width: 730px)");
+  const totalPosts = user.recipes?.length + user.reviews?.length;
+  const followers = user.followers?.length;
+  const following = user.following?.length;
+  return (
+    <div className="profile-header m-auto  w-full border-b pb-2">
+      {/* DESKTOP MENU */}
+      {isAboveMediumScreens && (
```

Comment:
This does not seem maintainable. Are you planning to do this for all components? There is a lot of duplicate code. Can you use these breakpoint prefixes instead? https://tailwindcss.com/docs/responsive-design

Reply:
wouldnt there be more code if I use tailwind?